tests/init.lua

```diff
@@ -24,5 +24,5 @@ end
 
 require("plenary.test_harness").test_directory("tests/specs", {
   minimal_init = "tests/minimal_init.lua",
-  sequential = true,
+  -- sequential = true,
```

Comment:
Sequential makes the output of tests not interleave. Any reason to change this?

Reply:
No, don't think so. I'll put it back.